occ/CMakeLists.txt

```diff
@@ -430,9 +341,7 @@ add_library(${OCCPLUGIN} SHARED
 
 target_link_libraries(${OCCPLUGIN} PUBLIC
     FairMQ::FairMQ
-    gRPC::grpc++
-    gRPC::grpc++_reflection
-    protobuf::libprotobuf
+    Occ-proto
```

Comment:
Is an additional linkage to a new .so the only way?

Reply:
It does not create new .so. Occ-proto is new object library, which as exactly designed for these purposes so the build structure is not polluted by additional shared/static libraries